### PR TITLE
Trucks overview

### LINKED ISSUE #10: Trucks Overview / Carousel
Screenshot 2023-02-15 at 14 55 18

Including responsive behaviour (becomes carousel)

### PR DESCRIPTION
**Pull request by bpauli**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #10

Test URLs:
- Before: https://main--vg-volvotrucks-us--hlxsites.hlx.page
- After: https://issue-10--vg-volvotrucks-us--hlxsites.hlx.page

### CODE REVIEW

File: blocks/trucks/trucks.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    width: 20px;
+    height: 20px;
+    text-align: center;
+ }
+
+ .trucks-wrapper ul.controls li.active button::before {
+    content: "\f192";
+ }
+
+@media screen and (max-width: 767px) {
```
Comment (hannessolo) [inline blocks/trucks/trucks.css:83]: From what I've seen so far the recommendation/best practice for franklin projects has been mobile-first css, with `min-width` queries for desktop styles. 

Not sure if others have opinions on this cc @msagolj @Buuhuu (eg. how does Milo do it?).

But I think its ok to leave it as `max-width` for here, its probably not worth rewriting the css.

**Review by dkuntze — DISMISSED** [review]

File: blocks/trucks/trucks.css (hunk 1 above)
Comment (bpauli) [reply blocks/trucks/trucks.css]: @hannessolo @dkuntze I switched to a mobile first approach. Can you please have another look to this PR? Thanks!

**Review by hannessolo — APPROVED** [review]
Code sieht gut aus.

Das einzige ist noch, aussen rum um den Block hats noch padding. Du könntest auf mobile bei `.section.carousel-container` das padding entfernen.

Screenshot 2023-02-23 at 10 17 55

**Comment by bpauli** [comment]
@hannessolo I removed the padding. Can you please approve the PR so it's getting merged. Thx!

**Review by hannessolo — APPROVED** [review]

**Pull request merged**